### ISSUE TITLE
[Adaptive Cards> FlightItinerary.json]: Passengers name is not defined inside 'dl' list tag.

### ISSUE THREAD

**Issue by vaishali1397** [microsoft/AdaptiveCards#8189]
### Target Platforms

UWP

### SDK Version

3.0.2210.26001

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS version: 22H2 (Build 25267.1000)
Browser: Edge Dev (110.0.1556.0))
Application: AdaptiveCardVisualizer
Application Version: 3.0.2210.26001
Screen reader: Narrator

**Repro Steps:**

1. Open the Adaptive Card Visualizer application.
2. Turn on the Narrator through Ctrl+ Windows+ Enter key and press caps lock +space bar key to turn on the scan mode.
3. Navigate to the 'FlightItinerary.json' through tab key and activate it.
4. Now, navigate to 'Passengers' name through up/down arrow key.
5. Observe the issue.

**Actual Result:**
'Passengers' name such as 'Sarah Hum', 'Jeremy Goldberg' and 'Evan Litwak' are not defined inside 'dl' list tag.
Screen reader is only announcing as **'Sarah Hum**'.

**Expected Result:**
'Passengers' name such as 'Sarah Hum', 'Jeremy Goldberg' and 'Evan Litwak' should be defined inside 'dl' list tag.
Screen reader should announce as '**1 Sarah Hum**'.

**User Impact:**
Screen reader user will get impacted as user will not be able to know that how many passengers names are present which create difficulty in making proper track of the content.

**Usable link:**
https://microsoft.sharepoint.com/:w:/r/sites/accessibility/_layouts/15/doc2.aspx?sourcedoc=%7B64DD7C5C-AA62-4D53-886E-BCAEE5341C01%7D&file=Guideline%20101%20Recommended%20Assistive%20Technology%20Tools%20for%20MAS%20Testing%20of%20Microsoft%20Digital%20Properties_v3.docx&action=default&mobileredirect=true&cid=e5999b53-8463-45ff-86b3-086ee5e5c931&isSPOFile=1&ovuser=72f988bf-86f1-41af-91ab-2d7cd011db47%2Cv-mohittyagi%40microsoft.com&wdOrigin=TEAMS-ELECTRON.p2p.bim&wdExp=TEAMS-CONTROL&wdhostclicktime=1658296286674

“Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-vaibgu%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400980259288%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=TrcBaCyAPKg0WAQfdSZJP2i9pghMEDg59i2NeUUs5EQ%3D&reserved=0) “

### Screenshots

Uploading Usable~ Passengers name is not define inside dl list tag.mp4…

### Card JSON

```JSON
NA
```
